innowiseTasks

```diff
@@ -22,3 +22,58 @@ select Accounts.account_id, abs(sum(Cards.balance) - Accounts.acc_balance) from
 join Cards on Accounts.account_id = Cards.account_id
 group by Accounts.account_id,Accounts.acc_balance 
 having abs(sum(Cards.balance) - Accounts.acc_balance) != 0
+
+--Task 4
+select count(Cards.card_id) as Counts, TStatus.status_title from Cards
+join Accounts on Cards.account_id = Accounts.account_id
+join Users on Accounts.user_id = Users.user_id
+join TStatus on Users.status_id = TStatus.status_id
+group by TStatus.status_title
+
+--Task 5 
+go
+create procedure Addmoney as
```

Comment:
We should always use transaction via using stored procedure.
https://www.mssqltips.com/sqlservertip/4897/handling-transactions-in-nested-sql-server-stored-procedures/